### DISCUSSION TITLE
Escape key to remove selection

### DISCUSSION THREAD

**Discussion by agneevX** [jesec/flood#259] [Ideas]
### Summary

Currently, the only way to deselect an item is to refresh the page entirely.

### Idea of implementation

The escape key could be used on devices with physical keyboards to remove the selected items.

I'm not entirely sure about the UX on touch devices, especially since the only way to refresh pages in iOS PWA is to kill and relaunch.

**Comment by jesec** [comment]
See also #166 . 

I don't think there is a practical need to fully deselect. Though, this may be a minor UX improvement.
